integrations/google-analytics/lib/index.js

```diff
@@ -989,6 +989,7 @@ GA.prototype.productListViewedEnhanced = function(track) {
       variant: item.properties().variant,
       price: item.price(),
       position:
+        item.properties().position ||
```

Comment:
I believe this won't work if `properties.product.position` is `0` or something different than a number. We need to check if the position field is valid first (positive integer), and use it if so.